app/graphql/types/tweet_type.rb

```diff
@@ -0,0 +1,6 @@
+class Types::TweetType < Types::BaseObject
+  graphql_name "MojoTweet"
+
+  field "id", Integer
+  field "created_at", String
```

Comment:
I'm still new-ish to GraphQL, but aren't you missing the contents of the tweet here?

Reply:
I originally had it this way as we planned on using the id to embed the tweets, but it didn't work out. In the other PR I added more of the contents (e.g. favorites, retweets, and text), but I can add in more of the fields if you think it would be better.